docs/content_management/taxonomy/taxonomy.md

```diff
@@ -4,15 +4,15 @@ description: A taxonomy uses tags to categorize and organize content
 
 # Taxonomy
 
-Taxonomies (**Tags**) allow you to organize content to make it easy for your site users to browse and to deliver content appropriate for them. It also enhances content search.
+Taxonomies (**Tags**) allow you to organize content to make it easy for your site users to browse and to deliver content appropriate for them. 
```

Comment:
While tags can strictly speaking be used in content search (via API), they are not available as search filters in the Back Office and that can be confusing.